src/components/course/CoursePageSidebar.vue

```diff
@@ -156,6 +157,9 @@
     computed: {
       isCurrentTerm() {
         return this.course.termId === this.$config.currentTermId
+      },
+      sendInviteAvailable() {
+        return this.get(this.course, 'room.capability') && this.course.instructors.length && !this.course.hasOptedOut
```

Comment:
@paulkerschen - You can use `this.$_.get(...)` and keep lodash aliases out of Utils mixin. Better?

Reply:
@johncrossman Thanks, I forgot we were doing it that way.